app/views/account/users/_users_table.html.slim

```diff
@@ -0,0 +1,13 @@
+tbody
```

Comment:
this is not correct usage,
in HTML <table> tag must have only 1 <tbody> tag, in your case <tbody> will be rendered for each record in collection when you'r using `render '', collection: @users, as: :user`

Reply:
yeah i removed it ))